ajax/libs/angular-css-injector/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "angular-css-injector",
+  "description": "angularJS service to load dynamically CSS files",
+  "version": "1.4.0",
+  "filename": "angular-css-injector.min.js",
+  "keywords": [
+    "angular",
+    "css",
+    "injector"
+  ],
+  "author": "Gabriel Delépine",
```

Comment:
Let's modify to `"Gabriel Delépine <yappli@github.com>"` and let it as the same with upstream.
https://github.com/GabrielDelepine/angular-css-injector/blob/main/bower.json#L7
Thanks